CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 ## Changed
 - external-site redirector now requires URLs to either be whitelisted or signed
 - Move logic for activity snippets out of template
+- Move `get_page_state_url` out of templatetags to avoid circular dependencies
```

Comment:
I think this might have snuck in [from 2523](https://github.com/cfpb/cfgov-refresh/pull/2523/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR20) but should have been removed again [by 2599](https://github.com/cfpb/cfgov-refresh/pull/2599/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eL20).